### ISSUE TITLE
P_???_0321_01 seems incorrect

### ISSUE THREAD

**Issue by jordig100** [3MFConsortium/test_suites#76]
The P_???_321_01.3mf files, seems to be incorrect.

3MF Builder, Magics, and Netfabb show it as inverted normals:
![image](https://user-images.githubusercontent.com/28712944/207940183-f116890b-47c7-4a02-9023-e6de972cc045.png)

On the other hand it contains the -1 in the transform with make it a negative determinant. This is forbidden in latest core spec version: https://github.com/3MFConsortium/spec_core/blob/master/3MF%20Core%20Specification.md#33-3d-matrices

However if we pace a 1 instead of the -1 in the transform, the applications still complain about it.

This seems to be a case for a negative test case.

In Test suite revision 1.34: 
Fixes for GitHub Test_Suite issues 65, 66. Modified positive test cases 321_01, 506_01, 304_04, and added negative test case 405_04.

But the test case 321_01 seems to be the same. Not sure what happens to the other test files.

 Please verify.

**Comment by JimZuber** [comment]
Yes, recent changes to the core specification disallowing negative determinant invalidate this test case. I moved this positive test case (P_???_0321_01) to a negative test case (N_???_416_03). The test specification and suites 1 through 6 have been updated on the develop branch. @Jordi, I will let you close this issue assuming you are happy with the change.
